catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -251,7 +251,13 @@ public void create() {
 
 	public void cloneSpriteAndAddToStage(Sprite cloneMe) {
 		Sprite copy = cloneMe.cloneForCloneBrick();
-		//copy.resetSprite(); // can I safely remove this?
+
+		copy.resetSprite();
+		Sprite originalSprite = ProjectManager.getInstance().getCurrentSprite();
+		ProjectManager.getInstance().setCurrentSprite(copy);
+		cloneMe.cloneLooks(copy);
+		ProjectManager.getInstance().setCurrentSprite(originalSprite);
+
 		copy.look.createBrightnessContrastHueShader();
```

Comment:
I __cannot test it__, but it LGTM for a hot-fix!

You could also move that code in the `Sprite.cloneForCloneBrick` itself so you don't have to care about the `getCurrentSprite` and `setCurrentSprite` but that's up to you.